src/components/JoinButton.jsx

```diff
@@ -0,0 +1,25 @@
+const JoinButton = () => {
+  // Inline style for the button
+  const buttonStyle = {
```

Comment:
use tailwindCSS instead of pure CSS

https://acm-ucr.github.io/docusaurus/docs/category/tailwind